index.md

```diff
@@ -4,27 +4,30 @@ NASA Astrophysics is developing the Fornax Initiative, a cloud-based system that
 brings together data, software, and computing so that researchers can focus on science.
 
 
-## Tutorial Notebooks
+## User Documentation
 
 ```{toctree}
 ---
 maxdepth: 2
-caption: Tutorial Notebooks
+caption: User Documentation
 ---
 
-forced_photometry/README
-light_curves/README
-```
+documentation/README
 
+```
 
-## User Documentation
+## Tutorial Notebooks
 
 ```{toctree}
 ---
 maxdepth: 2
-caption: User Documentation
+caption: Tutorial Notebooks
 ---
 
-documentation/README
+forced_photometry/README
+light_curves/README
+```
+
+Note that we store these notebooks as markdown files.  In the Fornax Science Console, these should open automatically in the runnable jupyter notebook format.  If they do not, or you run them elsewhere, you may need to convert them explicitly with 
```

Comment:
> "runnable jupyer notebook format" 

the myst-markdown is a totally legit jupter notebook format, so this will need to be rephrased. And they don't open up, the solution should not be the conversion but to sort out dependencies, or right click and open them up that way, explicitly as notebook.

(We can have the command half hidden somewhere, but I would rather advocate for how to use these properly and stop sending ipynb formats around).